packages/react-data-grid/src/Cell.js

```diff
@@ -297,6 +303,14 @@ const Cell = React.createClass({
     }
   },
 
+  removeScroll() {
+    let node = ReactDOM.findDOMNode(this);
```

Comment:
@diogofcunha i'm not sure of another way to do this. if you have any good ideas, please let me know.

I was only following the pattern used in setScrollLeft: https://github.com/adazzle/react-data-grid/blob/9d0e0bff012f9cdedf21c87a19314d2e14bee2e1/packages/react-data-grid-addons/src/draggable/DraggableHeaderCell.js#L18